src/dotnet.ts

```diff
@@ -1,32 +1,71 @@
+import * as path from 'path';
 import * as core from '@actions/core';
 import { exec } from '@actions/exec';
 import { ToolsDirectory } from './toolsDirectory';
 import { Platform } from './platform';
+import { fstat } from 'fs';
 
 const dotnetToolInstall = 'dotnet tool install';
+const dotnetToolUnInstall = 'dotnet tool uninstall';
 const dotnetCake = 'dotnet-cake';
 
 export class DotNet {
   static disableTelemetry() {
     core.exportVariable('DOTNET_CLI_TELEMETRY_OPTOUT', '1');
   }
 
-  static async installLocalCakeTool(targetDirectory: ToolsDirectory = new ToolsDirectory()) {
-    const cakeTool = Platform.isWindows() ? `${dotnetCake}.exe` : dotnetCake;
+  static async installLocalCakeTool(
+    targetDirectory: ToolsDirectory = new ToolsDirectory(),
+    version?:string
```

Comment:
Missing a space before the type.

```suggestion
    version?: string
```

Reply:
addressed.